web-ui/src/components/personnel/Personnel.jsx

```diff
@@ -46,8 +49,9 @@ const PersonnelComponent = () => {
     updateCheckins();
   },[personnel]);
 
-  function createEntry(person, checkIn) {
-    let key = undefined;
+  // Create entry of member and their last checkin
+  function createEntry(person, checkIn, key) {
+    key = key ? key : undefined;
```

Comment:
Every time I see this (even when I've written it myself) it makes me tilt my head to the side a little.

Reply:
Changed the key arg, so its more obvious.